packages/vde2/build.sh

```diff
@@ -0,0 +1,16 @@
+TERMUX_PKG_HOMEPAGE=https://github.com/virtualsquare/vde-2
+TERMUX_PKG_DESCRIPTION="Virtual Distributed Ethernet for emulators like qemu"
+TERMUX_PKG_LICENSE="custom"
```

Comment:
Package seem to have three different licenses. Use custom, gpl-2.0, lgpl-2.1 (that's what [archlinux does](https://archlinux.org/packages/extra/x86_64/vde2/)) and set LICENSE_FILE to all three COPYING files

Reply:
If package uses custom license, even besides GPL, LGPL or other well known, there should be specified "custom".

Reply:
Why? To me it looks like different part of the package have different licenses, most of it is packaged under GPL, but they have chose to license the shared libraries under LGPL, and the program slirpvde under a custom (modified BSD 3-clause) license. However, looks like slirpvde is linked against another GPL library in their source, so I suppose the GPL should apply to slirpvde as well (but I am not a lawyer or expert in licensing).

Specifying all three licenses (GPL, LGPL, custom), and packaging all three Copying files, gives the user the best idea about the licensing-status of the package IMO

Reply:
Ok, only custom needed to pass lint script